### ISSUE TITLE
Compare string to number

### ISSUE THREAD

**Issue by poboy975** [OmgFyiTbh/farmADV#2]
Hello, I tried your program today, and I got this error when starting. inv:108 attempt to compare string to number and the program stopped after that.

Minecraft 1.18
CC Tweaked cc-tweaked-1.18-1.99.0

**Comment by OmgFyiTbh** [comment]
Okay that's interesting never had that before. I must admit I haven't played since I uploaded this last year I'll take a look. Did the program work at all? What where the conditions of the turtle is did you already fuel it?

**Comment by poboy975** [comment]
I didn't fuel it, I have the fuel need turned off in my world. I can throw some coal in it and try it again.

*update* I put half a stack of coal into the turtle, and tried starting the farmADV. Same error happened, and the coal vanished. Didn't drop on the ground or get put into any chest. Just gone. Might be a side effect of having the fuel need turned off.

**Comment by OmgFyiTbh** [comment]
Yeah, coal will just disappear when the turtle fuels itself. I'm just firing up FTB to try it out not touched it in ages.

**Comment by OmgFyiTbh** [comment]
So I've tested the program in FTB revelations and its working fine at least to the level it always did... Can I suggest you try again with fuel requirement turned on. I can't see where to disable fuel so can't test it myself. Please let me know.

**Comment by poboy975** [comment]
Ok, sorry for the delay. I turned the fuel need back on, and it worked. I also have a tree farm turtle running, which worked fine without fuel, here is the pastebin for the tree farm. It might have the answers to the unlimited setting for fuel needs.
https://pastebin.com/93jEmv52
